src/foam/parse/NewlineParser.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.parse;
+
+import foam.lib.parse.Alt;
+import foam.lib.parse.Literal;
+import foam.lib.parse.Parser;
+
+public class NewlineParser {
+  public static Parser create() {
+    return new Alt(Literal.create("\n"), Literal.create("\r\n"));
```

Comment:
can add "\r" to new line parser
```suggestion
    return new Alt(Literal.create("\n"), Literal.create("\r\n"), Literal.create("\r"));
```

Reply:
Do we need to support CR line endings in FScript? @kgrgreer @penzital